packages/core/src/cart/headless-cart/map-to-cart.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { Cart } from '@bigcommerce/checkout-sdk/payment-integration-api';
+
+import { getCart } from '../carts.mock';
+
+import mapToCart from './map-to-cart';
+import { getHeadlessCartResponse } from './mocks/headless-cart.mock';
+
+describe('mapToCart', () => {
+    let headlessCartResponse: Cart | undefined;
+
+    beforeEach(() => {
+        headlessCartResponse = mapToCart(getHeadlessCartResponse().data.site);
+    });
+
+    it('maps to internal cart', () => {
+        const cart = getCart();
+
+        // TODO:: data is not yet fully compatible due to lack of information
```

Comment:
When do this information will be available?

Reply:
Need to update graphql request with providing this information. All missing information are not required for current implementation, so we can get rid of it by setting default values